src/components/BorrowerProfile/LendCta.vue

```diff
@@ -153,6 +153,12 @@
 							{{ ctaButtonText }}
 						</kv-ui-button>
 					</span>
+					<borrower-social-share
+						v-show="socialShareExp"
```

Comment:
Can we use `v-if` instead of `show` here?

Reply:
Same with the other include below

Reply:
Yeah! I forgot about the conditional display with `v-show`